internal/app/tfsec/debug/debug.go

```diff
@@ -0,0 +1,16 @@
+package debug
+
+import (
+	"fmt"
+	"time"
+)
+
+var Enabled bool
+
+func Log(format string, args ...interface{}) {
+	if !Enabled {
+		return
+	}
+	line := fmt.Sprintf(format, args...)
```

Comment:
What do you think about using runtime.Caller to get the calling function?

Might be a useful detail to include in the log line?

Reply:
Yeah, good idea. It'll be worth it as we begin to add debug lines everywhere...